### PR TITLE
Add: kubelet will evict kube/system processes

### PR DESCRIPTION
**Pull request by guettli**
It took me some time to understand the text. Maybe these
two sentences wil make it more clear.

Related issue: #32622

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ kubernetes system daemons.
 To optionally enforce `kube-reserved` on kubernetes system daemons, specify the parent
 control group for kube daemons as the value for `--kube-reserved-cgroup` kubelet
 flag.  If `kube-reserved` is enforced, kubelet will evict processes 
-of the specified cgroup if the thresholds are exceeded.
+of the specified cgroup if a threshold is exceeded.
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/reserve-compute-resources.md:90]: "if the threshold is exceeded" is more accurate?

Comment (guettli) [reply content/en/docs/tasks/administer-cluster/reserve-compute-resources.md]: I updated the text (twice)

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 2]
```diff
@@ -114,7 +115,8 @@ daemons.
 
 To optionally enforce `system-reserved` on system daemons, specify the parent
 control group for OS system daemons as the value for `--system-reserved-cgroup`
-kubelet flag.
+kubelet flag. If `system-reserved` is enforced, kubelet will evict processes 
+of the specified cgroup if the threshold is exceeded.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/reserve-compute-resources.md:119]: What does “evict” mean here?

**Review by sftim — COMMENTED** [review]
/sig node
A question about the intended meaning.

**Comment by divya-mohan0209** [comment]
FYI - @kubernetes/sig-node-pr-reviews , please could you clarify?

**Comment by sftim** [comment]
@guettli there's an open question about kubelet evicting processes - what happens in detail during this eviction? Is it in fact a Pod-level eviction, rather than at the process level?

**Comment by guettli** [comment]
> @guettli there's an open question about kubelet evicting processes - what happens in detail during this eviction? Is it in fact a Pod-level eviction, rather than at the process level?

@sftim sorry, I have no clue. I am new to Kubernetes. Where could I ask to find an answer to your question?

**Comment by zacharysarah** [comment]
@guettli 👋🏻 You can ask on [Kubernetes Slack](https://slack.kubernetes.io/), in the [#sig-node channel](https://app.slack.com/client/T09NY5SBT/C0BP8PW9G).

**Comment by divya-mohan0209** [comment]
Hello @guettli , good day! A gentle reminder, would you be willing to request SIG Node folks for an answer for further clarification?

**Comment by guettli** [comment]
@divya-mohan0209 I asked in the sig-node slack channel some time ago, but there was no proposal for a matching term. I am fine with closing this issue if nobody has a proposal. I am missing the right wording for a PR. The kublet itself is not killing any processes. I just specifies the requirements, and leaves to action up to the pperating system.

**Comment by divya-mohan0209** [comment]
Hello @guettli , thank you for your response! I've reiterated our request and am awaiting a response as well.

**Comment by kbhawkey** [comment]
👀

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md (hunk 2 above)
Comment (endocrimes) [reply content/en/docs/tasks/administer-cluster/reserve-compute-resources.md:119]: The kubelet won't "evict" processes - the system will starve them of CPU/OOMKill them to keep them within limits.

It does this by setting limits on the existing cgroups to whatever kube-reserved/system-reserved are set to (https://github.com/kubernetes/kubernetes/blob/3ffdfbe286ebcea5d75617da6accaf67f815e0cf/pkg/kubelet/cm/node_container_manager_linux.go#L114-L133).

**Review by endocrimes — CHANGES_REQUESTED** [review]

**Comment by sftim** [comment]
Thanks @endocrimes 

@guettli do you have what you need to know to move this PR forward?

**Comment by mehabhalodiya** [comment]
@guettli any updates on this?

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md (hunk 2 above)
Comment (reylejano) [reply content/en/docs/tasks/administer-cluster/reserve-compute-resources.md:119]: If the kubelet won't evict processes, I suggest this sentence be re-phrased

**Comment by divya-mohan0209** [comment]
Thank you @endocrimes ! 

@guettli Please could you check the suggestions provided by sig node reviewers & incorporate them?

**Comment by kbhawkey** [comment]
Hi @guettli .
Since this pull request has been open for some time, I plan to close it.
You can reopen the PR and continue to work on your changes.
/close

**Pull request closed without merging**